packages/evmosjs/README.md

```diff
@@ -1,3 +1,309 @@
-# Evmosjs
+# evmosjs
```

Comment:
Can we link to the main repo README instead? This way we can avoid duplicating the same information

Reply:
Yeah, actually I would update the main repo README explaining that is a monorepo containing all the evmosjs packages. Then we can add there the link to the packages/evmosjs/README.md. In this way we can have the README on NPM that is useful for devs